legal-api/report-templates/amalgamationApplication.html

```diff
@@ -15,6 +15,13 @@
             [[logo.html]]
           </td>
           <td>
+            {% if amalgamationApplication.type == 'regular' %}
+              <div class="report-type">REGULAR FORM</div>
```

Comment:
@severinbeauvais do you know if this has to be `REGULAR` or `REGULAR FORM`?

Reply:
or REGULAR LONG FORM?

Reply:
That's a Mihai question.

Reply:
This is what I know:

![image](https://github.com/user-attachments/assets/a5abc11a-2bd4-4c72-8a16-0ccd9350ddd1)

Reply:
Checking with Mihai now. Pending.

Reply:
Confirmed: REGULAR FORM